java/src/main/java/com/google/appengine/tools/pipeline/impl/PipelineManager.java

```diff
@@ -81,6 +83,14 @@ public class PipelineManager implements PipelineRunner, PipelineOrchestrator {
 
   public static final String DEFAULT_QUEUE_NAME = "default";
 
+  @RequiredArgsConstructor
+  @Getter
+  public enum ConfigProperty implements com.google.appengine.tools.pipeline.util.ConfigProperty {
+    SHARDED_JOBS_DEFAULT_QUEUE("com.google.appengine.tools.pipeline.SHARDED_JOBS_DEFAULT_QUEUE");
```

Comment:
i think clearer to call `INCREMENTAL_TASK_DEFAULT_QUEUE`

Reply:
or worker, as that's what it is ...

the actual "job" invocations - eg, orchestration of the pipeline - will still happen through the other queues, right?